### DISCUSSION TITLE
Bug: priority, team, iteration fields not accessible

### DISCUSSION THREAD

**Discussion by mesqueeb** [community/community#11830] [Projects and Issues] [answered]
Both in these two blog posts now:
- https://github.blog/changelog/2021-12-02-the-new-github-issues-12-02-update/#💫-view-your-boards-by-any-field
- https://github.blog/changelog/2022-02-02-the-new-github-issues-02-02-update/#🌀-flexible-iterations

There is mention of
- priority
- team
- iteration

special fields. while we only have the "status" and don't seem to have access to these other fields, even though we are using the issues beta project.

Any idea how to turn on these new fields?

This is what I see when I look at the project settings:

<img width="646" alt="image" src="https://user-images.githubusercontent.com/3253920/154869157-fa837a90-2620-462d-a678-c822433ff14a.png">

Where do I add the priority fields?

**Comment by pedrompeixoto** [comment, marked as answer]
Hello @mesqueeb,

You might need to add an iteration column to your project.
Click the plus icon in the table header and add the iteration column.

<img width="1510" alt="Screenshot 2022-02-21 at 00 47 32" src="https://user-images.githubusercontent.com/91019696/154872530-d4784505-a839-4e60-a197-27ea20556f35.png">
